elements/rh-announcement/demo/rh-announcement.html

```diff
@@ -0,0 +1,42 @@
+<link rel="stylesheet" href="demo.css">
+
+<rh-announcement color-palette="dark">
+    <img src="https://placehold.co/120x40?text=logo" slot="image" alt="LOGO" width="120" height="40">
+    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit adipisicing elit adipisicing elit.</p>
+    <rh-cta slot="cta">
+        <a href="#" data-analytics="something">Learn More</a>
+    </rh-cta>
+</rh-announcement>
+
+<rh-announcement imgleft>
+  <img src="https://placehold.co/40x40?text=icon" slot="image" alt="ICON" width="40" height="40">
+  <p>Lorem ipsum dolor sit amet consectetur elit.</p>
+  <rh-cta slot="cta">
+      <a href="#" data-analytics="something">Learn More</a>
+  </rh-cta>
+</rh-announcement>
+
+<rh-announcement dismissable>
+  <img src="https://placehold.co/120x40?text=logo" slot="image" alt="LOGO" width="120" height="40">
+  <p>Lorem ipsum dolor sit amet consectetur adipisicing elit ipsum dolor sit. Lorem adipisicing elit.</p>
+  <rh-cta slot="cta">
+      <a href="#" data-analytics="something">Learn More</a>
+  </rh-cta>
+</rh-announcement>
+
+
+<rh-announcement color-palette="dark" imgleft dismissable>
+<img src="https://placehold.co/40x40?text=icon" slot="image" alt="ICON" width="50" height="40">
+<p>Lorem ipsum dolor sit amet consectetur adipisicing elit ipsum dolor sit amet consectetur adipisicing elit.</p>
+<rh-cta slot="cta">
+    <a href="#" data-analytics="something">Learn More</a>
+</rh-cta>
+</rh-announcement>
+
+
+
+
+<script type="module">
+  import '@rhds/elements/rh-cta/rh-cta.js';
+  import '@rhds/elements/rh-announcement/rh-announcement.js';
+</script>
```

Comment:
Probably a good idea to implement an `events.html` demo that shows how to implement `close`/`AnnouncementCloseEvent`. We usually demo this with [`window.confirm`](https://developer.mozilla.org/en-US/docs/Web/API/Window/confirm).

[`rh-video` events demo example](https://github.com/RedHat-UX/red-hat-design-system/blob/feat/video/elements/rh-video/demo/require-consent.html)